runtime/android/core_internal/src/org/xwalk/core/internal/XWalkContent.java

```diff
@@ -713,6 +713,26 @@ public void scrollBy(int x, int y) {
         mContentView.scrollBy(x, y);
     }
 
+    public int computeHorizontalScrollRange() {
```

Comment:
Since it's not easy for testing Scroll related API, I'd like to use an sample for testing.